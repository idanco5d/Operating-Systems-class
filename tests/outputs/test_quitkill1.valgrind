==2325== Memcheck, a memory error detector
==2325== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==2325== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==2325== Command: /mnt/c/Users/illus/os/hw/1/smash
==2325== Parent PID: 2320
==2325== 
==2325== 
==2325== FILE DESCRIPTORS: 4 open (3 std) at exit.
==2325== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_quitkill1.valgrind
==2325==    <inherited from parent>
==2325== 
==2325== 
==2325== HEAP SUMMARY:
==2325==     in use at exit: 0 bytes in 0 blocks
==2325==   total heap usage: 42 allocs, 42 frees, 80,525 bytes allocated
==2325== 
==2325== All heap blocks were freed -- no leaks are possible
==2325== 
==2325== For lists of detected and suppressed errors, rerun with: -s
==2325== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
