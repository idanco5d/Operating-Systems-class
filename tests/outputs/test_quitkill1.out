smash> smash> smash> smash> smash: sending SIGKILL signal to 1 jobs:
9588: sleep 100000&
